### PR TITLE
Corrected bug in calculation of restart date for ofp.

### PR DESCRIPTION
**Pull request by ashalper-usgs**

### CODE REVIEW

**Comment by ashalper-usgs** [comment]
The "correct dates" part is what I'm not sure of. Does this look correct to you?
```
 /opt/conda/bin/python -u /usr/local/src/onhm-fetcher-parser/pkg/climate_etl.py -t date -p 2019-07-14 2019-10-07 -i /var/lib/nhm/ofp/nhm_hru_data -o /var/lib/nhm/NHM-PRMS_CONUS//input -w /var/lib/nhm/ofp/nhm_hru_data/weights.csv
```

**Comment by rmcd-mscb** [comment]
Yes! those dates look correct

**Comment by ashalper-usgs** [comment]
OK, I'll merge it.

File: nhmusgs-base/nhm.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ last_simulation_date () {
   # Determine the date for the last simulation by finding the last
   # restart file.
   ls $dir$restart_dir*.restart | sed 's/^.*\///;s/\.restart$//' | \
```
Comment (isuftin) [inline nhmusgs-base/nhm.sh]: One of the issues we saw in the last run is that the directory structure was attached to the file name going into this command, so the sed command didn't really work out. 

This seems to work?

```
$ pwd
/tmp
$ mkdir -p dir1/dir2/dir_3
$ touch  dir1/dir2/dir_3/10-01-2019.restart
$ touch  dir1/dir2/dir_3/10-02-2019.restart
$ dir=`pwd`
$ restart_dir='/dir1/dir2/dir_3/'
ls $dir$restart_dir*.restart | sed 's/^.*\///;s/\.restart$//' | sort | tail -1
10-02-2019
```

Comment (ashalper-usgs) [reply nhmusgs-base/nhm.sh]: The directory path was coming from the first `ls` at line 17. The `> /dev/null 2>&1` bit in the patch omits that.

**Pull request merged**